packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -57,6 +58,8 @@ export const defaultExecutionEngineHttpOpts: ExecutionEngineHttpOpts = {
   timeout: 12000,
 };
 
+const QUEUE_MAX_LENGHT = 256;
```

Comment:
Write a JSDoc comment reasoning the choice of this number. What the tradeoff here? A job queue holds items in memory which in this case are full de-serialized blocks. Are there any memory concerns if the queue fills? What's a reasonable length given the usage of this queue? Since blocks are imported strictly sequential the queue length should always be pretty short.

Reply:
Will check from the blocks processor queue what limit it has, ideally there could be fcUs corresponding to the last batch of blocks plus new payload